From 828fabe949860de3dd5ff1105af2f76499758ed1 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Mon, 27 Jun 2016 17:04:54 +0800
Subject: [PATCH] soc: rockchip: power-domain: fix up the PMU_GPU_PWRDW/UP_CNT
 for RK3399

According to the advice of the IC,
setting the PMU_GPU_PWRDW/PWRUP_CNT regs 6 cycel(250ns) for RK3399 SOC.

Change-Id: I0449069a3b5035bd0442fcd74b645de9480a1d89
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/soc/rockchip/pm_domains.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/soc/rockchip/pm_domains.c b/drivers/soc/rockchip/pm_domains.c
index bd1f1c4e5f80..65a5a6d22c71 100644
--- a/drivers/soc/rockchip/pm_domains.c
+++ b/drivers/soc/rockchip/pm_domains.c
@@ -736,11 +736,11 @@ static const struct rockchip_pmu_info rk3399_pmu = {
 	.idle_offset = 0x64,
 	.ack_offset = 0x68,
 
-	.core_pwrcnt_offset = 0x9c,
-	.gpu_pwrcnt_offset = 0xa4,
+	.core_pwrcnt_offset = 0xac,
+	.gpu_pwrcnt_offset = 0xac,
 
-	.core_power_transition_time = 24,
-	.gpu_power_transition_time = 24,
+	.core_power_transition_time = 6, /* 0.25us */
+	.gpu_power_transition_time = 6, /* 0.25us */
 
 	.num_domains = ARRAY_SIZE(rk3399_pm_domains),
 	.domain_info = rk3399_pm_domains,
-- 
2.35.3

